tinygrad/tensor.py

```diff
@@ -152,7 +152,11 @@ def zeros(*shape, **kwargs): return Tensor.full(argfix(*shape), 0, **kwargs)
   def ones(*shape, **kwargs): return Tensor.full(argfix(*shape), 1, **kwargs)
 
   @staticmethod
-  def arange(stop, start=0, step=1, **kwargs): return Tensor.full((ceil((stop-start)/step),), step, **kwargs).cumsum() + (start - step)
+  def arange(start=0, stop=None, step=1, **kwargs):
```

Comment:
Can we make `start` required?

Reply:
ah yes good point. 

(sorry being really slow on this pr)